showcase.json

```diff
@@ -619,6 +619,21 @@
     "videoContentType": "video/mp4",
     "screenshots": []
   },
+  {
+    "id": "jumpymotoko",
+    "name": "Jumpy Motoko",
+    "website": "https://65t4u-siaaa-aaaal-qbx4q-cai.ic0.app/",
+    "tags": [
+      "Games"
+    ],
+    "description": "Unity play to earn game on Internet Computer. Comes with code sample so you can deploy your own Unity play to earn game too. https://github.com/therealbryanho/IC-Code-Sample-Unity-Play-to-Earn-Game",
```

Comment:
Hey @therealbryanho, thank you for the contribution. You can add the github link directly:

```js
"github":  "https://github.com/therealbryanho/IC-Code-Sample-Unity-Play-to-Earn-Game"
```

Reply:
Thanks! Have done the change.